datatable/readXlsxFromAttachment-test.roxtest.ts

```diff
@@ -26,16 +26,18 @@ describe("services", () => {
         const result = await readXlsxFromAttachment(environment);
         expect(result).to.equal(true);
         const resultField = environment.instanceDetails.extras.fieldContents["Büros"];
-        expect(resultField!.type).to.equal("ProcessHubDataTable");
-        const fieldValue = resultField!.value as IDataTableFieldValue;
-        expect(fieldValue).to.not.be.undefined;
-        expect(fieldValue.rows.length).to.equal(20);
-        for (const row of fieldValue.rows) {
-          expect(row.selected).to.equal(false);
-          expect(typeof row.data.Name).to.equal("string");
-          expect(typeof row.data.Adresse).to.equal("string");
-          expect(typeof row.data.Telefonnummer).to.equal("string");
-          expect(typeof row.data.Kostenstelle).to.equal("number");
+        if (resultField) {
```

Comment:
Why is this necessary? The non-null assertion should be allowed here as its a test file.

Reply:
yes my bad, this change was made before i excluded roxtest files from the eslint config